CMakeLists.txt

```diff
@@ -81,9 +81,23 @@ endif ()
 
 option (ENABLE_UNIT_TESTS "Enable unit tests" OFF)
 if (ENABLE_UNIT_TESTS)
-    find_package (GTest REQUIRED HINT $ENV{GTEST_PREFIX})
+    include(FetchContent)
+    include(GoogleTest)
+
+    enable_testing()
+
+    FetchContent_Declare(
+        googletest
+        # currently v1.14
+        URL https://github.com/google/googletest/archive/f8d7d77c06936315286eb55f8de22cd23c188571.zip
```

Comment:
Wouldn't it be better to use a link with the version, i.e. `https://github.com/google/googletest/archive/refs/tags/v1.14.0.zip`? One could then also provide the version as an optional argument to CMake.

Reply:
Very valid point, this is more expressive and directly clear to read.
Added in https://github.com/IPPL-framework/ippl/pull/233/commits/c8b366fc89229bf92821c0ccae03b0072ea0f220.